src/symbols.js

```diff
@@ -437,6 +437,18 @@ defineSymbol(math, main, textord, "\u2018", "`");
 defineSymbol(math, main, textord, "$", "\\$");
 defineSymbol(text, main, textord, "$", "\\$");
 defineSymbol(text, main, textord, "$", "\\textdollar");
+defineSymbol(math, main, textord, "€", "\\€");
+defineSymbol(text, main, textord, "€", "\\€");
+defineSymbol(text, main, textord, "€", "\\texteuro");
+defineSymbol(math, main, textord, "¥", "\\¥");
+defineSymbol(text, main, textord, "¥", "\\¥");
+defineSymbol(text, main, textord, "¥", "\\textyen");
+defineSymbol(math, main, textord, "₹", "\\₹");
+defineSymbol(text, main, textord, "₹", "\\₹");
+defineSymbol(text, main, textord, "₹", "\\textrupee");
+defineSymbol(math, main, textord, "₽", "\\₽");
+defineSymbol(text, main, textord, "₽", "\\₽");
+defineSymbol(text, main, textord, "₽", "\\textruble");
```

Comment:
I thought you'd be adding:
```
\textbar
\textbardbl
\textbraceleft
\textbraceright
\textdagger
\textdaggerdbl
\textdollar
\textellipsis
\textemdash
\textendash
\textgreater
\textless
\textquotedblleft
\textquotedblright
\textquoteleft
\textquoteright
\textsterling
\textunderscore
```
which we already have symbols for.  Unfortunately, we don't have any fonts with with glyphs for the Euro, Rupee, or Ruble.  The Yen is already supported.  Also, defining symbols with unicode support should look like:
```
defineSymbol(math, ams, textord, "\u00a5", "\\yen", true);
```